e2e/specs/drug-orders.spec.ts

```diff
@@ -1,228 +1,283 @@
-  });
-
-  await test.step('And I click on the "Save Order" button', async () => {
-    await page.getByRole('button', { name: /save order/i }).click();
-  });
-
-  await test.step('Then the order status should be changed to "Modify"', async () => {
-    await expect(orderBasket.getByText(/new/i)).toBeHidden();
-    await expect(orderBasket.getByText(/modify/i)).toBeVisible();
-  });
-
-  await test.step('When I click on the "Sign and close" button', async () => {
-    await page.getByRole('button', { name: /sign and close/i }).click();
-  });
-
-  await test.step('Then I should see a success notification', async () => {
-    await expect(page.getByText(/updated aspirin 81mg/i)).toBeVisible();
-  });
-
-  await test.step('And I should see the updated order in the list in Active Medications table', async () => {
-    const headerRow = medicationsPage.medicationsTable().locator('thead > tr');
-    const dataRow = medicationsPage.medicationsTable().locator('tbody > tr');
-
-    await expect(headerRow.nth(0)).toContainText(/start date/i);
-    await expect(headerRow.nth(0)).toContainText(/details/i);
-    await expect(dataRow.nth(0)).toContainText(/aspirin 81mg/i);
-    await expect(dataRow.nth(0)).not.toContainText(/1 tablet/i);
-    await expect(dataRow.nth(0)).toContainText(/2 tablet/i);
-    await expect(dataRow.nth(0)).not.toContainText(/oral/i);
-    await expect(dataRow.nth(0)).toContainText(/inhalation/i);
-    await expect(dataRow.nth(0)).not.toContainText(/once daily/i);
-    await expect(dataRow.nth(0)).toContainText(/twice daily/i);
-    await expect(dataRow.nth(0)).not.toContainText(/3 days/i);
-    await expect(dataRow.nth(0)).toContainText(/5 days/i);
-    await expect(dataRow.nth(0)).not.toContainText(/indication headache/i);
-    await expect(dataRow.nth(0)).toContainText(/indication hypertension/i);
-  });
-
-  await test.step('When I click the overflow menu in the table row with the updated medication', async () => {
-    await page
-      .getByRole('button', { name: /options/i })
-      .nth(0)
-      .click();
-  });
-
-  await test.step('And I click on the "Discontinue" button', async () => {
-    await page.getByRole('menuitem', { name: /discontinue/i }).click();
-  });
-
-  await test.step('Then the order status should be changed to "Discontinue"', async () => {
-    await expect(orderBasket.getByText(/discontinue/i)).toBeVisible();
-  });
-
-  await test.step('And I click on the "Sign and close" button', async () => {
-    await page.getByRole('button', { name: /sign and close/i }).click();
-  });
-
-  await test.step('Then I should see a success notification', async () => {
-    await expect(page.getByText(/discontinued aspirin 81mg/i)).toBeVisible();
-  });
-
-  await test.step('And the medications table should be empty', async () => {
-    await expect(page.getByText(/there are no active medications to display for this patient/i)).toBeVisible();
+test.describe.serial('Drug Order Tests', () => {
```

Comment:
I think `serial` does exactly what we wanted to avoid. That is, if the "Record a drug test" fails, we should still run the "Edit a drug order". I'm also unsure about grouping the tests like this, as it now makes them rely on a single patient, whereas we should have a patient-per-test.

```suggestion
test.describe('Drug Order Tests', () => {
```